collections/nemo_tts/nemo_tts/parts/layers.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2019 NVIDIA Corporation
+import torch
+
+
+class LinearNorm(torch.nn.Module):
```

Comment:
why call this LinearNorm, isn't this just "Linear"?

Reply:
TBH, I have no idea, I took this from Deep Learning Examples. I think for future releases, I will be moving away from reusing code from Deep Learning Examples, and just git submodule the nvidia tacotron 2 and waveglow repos.